README.md

```diff
@@ -56,7 +56,7 @@ The `Tag` can have two valid values: `1` (`Msg_Data`) and `3` (`Msg_Error`). In
 
```

Comment:
Resolved in separate discussion. Not possible as `pip` only supports markdown and reStructuredText. Is there an upstream issue that we could star (or should we create one)?

Reply:
Being able to use AsciiDoc for all documentation would be nice. I'm not sure where the right place would be to create a feature request. I couldn't find an existing ticket. The limitation to Markdown and reStructuredText already starts at the creation of the source distribution (cf. https://packaging.python.org/guides/using-manifest-in/), although that is not a real issue, as a `README.adoc` could still be added manually to the package. The limitation for PyPI is documented here: https://packaging.python.org/guides/making-a-pypi-friendly-readme/.

Reply:
It created a [ticket for readme_renderer](https://github.com/pypa/readme_renderer/issues/205), even though I don't think it will be implemented anytime soon.